packages/network/src/connection/PeerInfo.ts

```diff
@@ -128,7 +128,19 @@ export class PeerInfo {
             longitude: null,
             country: null,
             city: null
+        } 
+
+        if (peerType === PeerType.Tracker){
+            this.peerId = peerId
+            return
+        }
+
+        if (peerId.indexOf('_sid_') >= 0){
+            this.peerId = peerId
+        } else {
+            this.peerId = `${peerId}_sid_${uuidv4()}`
```

Comment:
replaced `#` with `_sid_` as a split between the peerId and sessionId to allow the HTTP endpoints to work as expected